modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/macros/CustomerPortal.macro

```diff
@@ -265,6 +265,27 @@ definition {
 		CustomerPortal.showCPObject(title = "Koroneiki Account");
 	}
 
+	macro createUserForCP {
+		JSONUser.addUser(
```

Comment:
@luanakarla would it be better to let as default user set at `test.user@liferay.com` but let the option to change the user's parameters?

Reply:
Done